packages/clerk-js/src/core/resources/Environment.ts

```diff
@@ -58,6 +60,7 @@ export class Environment extends BaseResource implements EnvironmentResource {
   protected fromJSON(data: EnvironmentJSON | null): this {
     if (data) {
       this.authConfig = new AuthConfig(data.auth_config);
+      this.meta = data.meta;
```

Comment:
Lets keep this as it was before - env does not need to know about everything in meta, just the country
```suggestion
      this.country = data.meta.headers.country
```